content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,20 @@
+title: Russian's war on Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war
+status: draft
+comments: yes
+
+Since the 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. This requires that we all speak up and show solidarity for Ukraine.
+
+Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many young Russian soldiers who are sent by a dictator into a senseless war that they do not want, do not understand and are not even allowed to call it as such.
+
+Putin's propaganda, misinformation and punishment of the opposition has secured him some positive support among russian people. The extent of censorship and political persecution is unprecedented in the recent Russian history.
```

Comment:
I suggest to remove this paragraph and focus on the actual incidents.

Reply:
OK, I have removed it.